tests/api/api/test_datastore_profiles.py

```diff
@@ -0,0 +1,191 @@
+# Copyright 2023 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import json
+from http import HTTPStatus
+
+from fastapi.testclient import TestClient
+from sqlalchemy.orm import Session
+
+import mlrun.artifacts
+import mlrun.common.schemas
+
+project = "prj"
+datastore = {
+    "project": "prj",
+    "name": "ds",
+    "type": "nosql",
+    "body": "body or url ds://host:1234",
+}
+legacy_api_projects_path = "projects"
+api_datastore_path = "/api/v1/projects/{project}/datastore_profiles"
```

Comment:
```suggestion
api_datastore_path = f"/api/v1/projects/{project}/datastore_profiles"
```

Reply:
This is by design - testing special characters in the path

Reply:
In the context of this line, that doesn't really make sense. It's clearly a string substitution.

Reply:
I see that this gets substituted in each test, but the project name is always the same. Seems to me like it can be substituted here.